src/com/amazon/ion/benchmark/schema/constraints/Contains.java

```diff
@@ -0,0 +1,37 @@
+package com.amazon.ion.benchmark.schema.constraints;
+
+import com.amazon.ion.IonList;
+import com.amazon.ion.IonValue;
+
+/**
+ * Process the constraint 'Contains' which indicates that the list or S-expression is expected to contain all of specified
+ * values, in no particular order.
+ */
+public class Contains implements ReparsedConstraint {
+    private final IonList expectedContainedValue;
+
+    /**
+     * Initializing the newly created Object.
+     * @param expectedContainedValue represents the value of constraint 'contains'.
+     */
+    public Contains(IonValue expectedContainedValue) {
```

Comment:
Since there's a public factory method, the constructor should be private.

Reply:
> Since there's a public factory method, the constructor should be private.

Will update the constructors in factory pattern as internal in the following commit.